docs/src/opf/acp.md

```diff
@@ -1,3 +1,149 @@
 # AC-OPF
 
-See [`ACPPowerModel`](https://lanl-ansi.github.io/PowerModels.jl/stable/formulation-details/#PowerModels.ACPPowerModel) formulation in [`PowerModels.jl`](https://lanl-ansi.github.io/PowerModels.jl/stable/).
\ No newline at end of file
+
+## Mathematical Formulation
+
+The ACOPF model considered in OPFGenerator is presented below.
```

Comment:
Overall comment on the model: we've encountered some issues with the sign of dual variables for constraints that are written like
```julia
@constraint(model, x + y == z)
```
where `x, y, z` are all `JuMP.VariableRef`s. Namely, JuMP may end up with `x + y - z == 0` or `z - x - y == 0` (I assume there is some consistency in how that transformation is done, but I don't know what it is). We ended up with dual variables that were negated compared to what we expected :(

I wrote down all the constraints with variables on the left and constant terms on the right to avoid that risk.
Again, no impact on the primal, but it can change the sign of dual variables, so I'd like to have as clear conventions as possible.